jabgui/src/main/java/org/jabref/gui/mergeentries/MultiMergeEntriesView.java

```diff
@@ -126,13 +129,15 @@ public void initialize() {
         leftScrollPane.vvalueProperty().bindBidirectional(centerScrollPane.vvalueProperty());
         rightScrollPane.vvalueProperty().bindBidirectional(centerScrollPane.vvalueProperty());
 
-        viewModel.failedSuppliersProperty().addListener((obs, oldValue, newValue) ->
-                failedSuppliers.setText(viewModel.failedSuppliersProperty().get().isEmpty() ? "" : Localization.lang(
-                        "Could not extract Metadata from: %0",
-                        String.join(", ", viewModel.failedSuppliersProperty())
-                ))
-        );
-
+        viewModel.failedSuppliersProperty().addListener((obs, oldValue, newValue) -> {
```

Comment:
Since we are touching this class anyway, lets ignore some unused observables
```suggestion
        viewModel.failedSuppliersProperty().addListener((_, _, _) -> {
```